src/apps/sources/pages/CreateSourcePage.tsx

```diff
@@ -33,10 +35,15 @@ const CreateSourcePage: React.FC<Props> = ({
   errors,
   createSourceAction,
   loading,
+  resetCreateSource,
   newSource
   }: Props) => {
     const previouslyLoading = usePrevious(loading);
 
+    useEffect(() => resetCreateSource(),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []);
```

Comment:
```suggestion
    // eslint-disable-next-line react-hooks/exhaustive-deps
    useEffect(() => resetCreateSource, []);
```

I don't know why, but this seems to work...